apps/studio/components/interfaces/Organization/BillingSettings/PaymentMethods/PaymentMethods.tsx

```diff
@@ -76,7 +79,14 @@ const PaymentMethods = () => {
           </div>
         </ScaffoldSectionDetail>
         <ScaffoldSectionContent>
-          {!canReadPaymentMethods ? (
+          {selectedOrganization?.managed_by !== undefined &&
+          selectedOrganization?.managed_by !== 'supabase' ? (
+            <PartnerManagedResource
+              partner={selectedOrganization?.managed_by}
+              resource="Payment Methods"
+              ctaUrl={`https://vercel.com/${'supabase'}/~/settings/billing`}
```

Comment:
unnecessary interpolation?

```suggestion
              ctaUrl={`https://vercel.com/supabase/~/settings/billing`}
```

Reply:
These have all been fixed now 👍